### ISSUE TITLE
RELRO Checking is incorrect for crafted ELF file

### ISSUE THREAD

**Issue by disconnect3d** [etke/checksec.rs#2]
The tool misreports the issue reported here: https://github.com/slimm609/checksec.sh/issues/161

Maybe RELRO should be checked only for got and reported as "GOT RELRO"?

**Comment by etke** [comment]
@disconnect3d , Thanks for opening this issue! I will dig into the linked issue and further extension of the RELRO check to address efficacy concerns.

**Comment by etke** [comment]
Resolved in 0.0.7 Release with commit https://github.com/etke/checksec.rs/commit/cdb136df587222f23d8ad7d2ebf14ae45ec29624

Default behavior will return Partial RELRO if `PT_GNU_RELRO`  header is present unless both `DF_BIND_NOW` and `DF_1_NOW` flags are set.

```sh
ELF64: | Canary: false CFI: false SafeStack: false Fortify: false Fortified: 0 NX: true PIE: None Relro: Partial RPATH: None RUNPATH: None | File: ./easiest_patch
```
